pkg/api/folder.go

```diff
@@ -807,7 +807,14 @@ func (fk8s *folderK8sHandler) writeError(c *contextmodel.ReqContext, err error)
 	//nolint:errorlint
 	statusError, ok := err.(*k8sErrors.StatusError)
 	if ok {
-		c.JsonApiErr(int(statusError.Status().Code), statusError.Status().Message, err)
+		message := statusError.Status().Message
+		// #TODO: Is there a better way to set the correct meesage? Instead of "access denied to folder", currently we are
+		// returning something like `folders.folder.grafana.app is forbidden: User "" cannot create resource "folders" in
```

Comment:
I'm not sure if this has to do with the authorizer? If so, we could change the error message there instead

Reply:
Yeah, perhaps there is something that can be updated in the authorizer. I don't think the message is being emitted there though 🤔  I'll add this as a task because it might take a bit more work/investigation and I want to fix the UI sooner rather than later. Thanks for the review!